tests/cross_fw/shared/helpers.py

```diff
@@ -19,15 +19,20 @@
 
 import numpy as np
 
-import nncf
-from nncf.common.utils.os import is_linux
-from nncf.common.utils.os import is_windows
 from tests.cross_fw.shared.paths import GITHUB_REPO_URL
 from tests.cross_fw.shared.paths import PROJECT_ROOT
 
 TensorType = TypeVar("TensorType")
 
 
+def is_windows() -> bool:
+    return "win32" in sys.platform
+
+
+def is_linux() -> bool:
+    return "linux" in sys.platform
```

Comment:
The same comment about just copying `nncf.common.utils.os.is_linux` and `nncf.common.utils.os.is_windows`.